pkg/bpf/bpf_kmesh_l4.go

```diff
@@ -58,7 +58,7 @@ func (sc *BpfKmesh) ApiEnvCfg() error {
 	var info *ebpf.MapInfo
 	var id ebpf.MapID
 
-	info, err = sc.SockConn.KmeshCgroupSockMaps.KmeshListener.Info()
+	info, err = sc.SockConn.GetKmeshListenerMap().Info()
```

Comment:
There is a question of whether we only need to distinguish versions when dealing with prog, and map does not.

Reply:
Both progs and maps generated by `bpf2go` are in 2 different structs, here is an example:

**Hi version:**
```
type KmeshCgroupSockHiObjects struct {
	KmeshCgroupSockHiPrograms
	KmeshCgroupSockHiMaps
}

type KmeshCgroupSockHiMaps struct {
	BpfLogLevel         *ebpf.Map `ebpf:"bpf_log_level"`
	InnerMap            *ebpf.Map `ebpf:"inner_map"`
	KmeshCluster        *ebpf.Map `ebpf:"kmesh_cluster"`
	KmeshEvents         *ebpf.Map `ebpf:"kmesh_events"`
	KmeshListener       *ebpf.Map `ebpf:"kmesh_listener"`
	KmeshManage         *ebpf.Map `ebpf:"kmesh_manage"`
	KmeshTailCallCtx    *ebpf.Map `ebpf:"kmesh_tail_call_ctx"`
	KmeshTailCallProg   *ebpf.Map `ebpf:"kmesh_tail_call_prog"`
	MapOfClusterEps     *ebpf.Map `ebpf:"map_of_cluster_eps"`
	MapOfClusterEpsData *ebpf.Map `ebpf:"map_of_cluster_eps_data"`
	MapOfSockStorage    *ebpf.Map `ebpf:"map_of_sock_storage"`
	OuterMap            *ebpf.Map `ebpf:"outer_map"`
	TmpBuf              *ebpf.Map `ebpf:"tmp_buf"`
	TmpLogBuf           *ebpf.Map `ebpf:"tmp_log_buf"`
}
```

**Lo version:**
```
type KmeshCgroupSockLoObjects struct {
	KmeshCgroupSockLoPrograms
	KmeshCgroupSockLoMaps
}

type KmeshCgroupSockLoMaps struct {
	BpfLogLevel         *ebpf.Map `ebpf:"bpf_log_level"`
	InnerMap            *ebpf.Map `ebpf:"inner_map"`
	KmeshCluster        *ebpf.Map `ebpf:"kmesh_cluster"`
	KmeshEvents         *ebpf.Map `ebpf:"kmesh_events"`
	KmeshListener       *ebpf.Map `ebpf:"kmesh_listener"`
	KmeshManage         *ebpf.Map `ebpf:"kmesh_manage"`
	KmeshTailCallCtx    *ebpf.Map `ebpf:"kmesh_tail_call_ctx"`
	KmeshTailCallProg   *ebpf.Map `ebpf:"kmesh_tail_call_prog"`
	MapOfClusterEps     *ebpf.Map `ebpf:"map_of_cluster_eps"`
	MapOfClusterEpsData *ebpf.Map `ebpf:"map_of_cluster_eps_data"`
	MapOfSockStorage    *ebpf.Map `ebpf:"map_of_sock_storage"`
	OuterMap            *ebpf.Map `ebpf:"outer_map"`
	TmpBuf              *ebpf.Map `ebpf:"tmp_buf"`
	TmpLogBuf           *ebpf.Map `ebpf:"tmp_log_buf"`
}
```

As you can see, maps in `KmeshCgroupSockHiMaps` and `KmeshCgroupSockLoMaps` are totally the same, but  `KmeshCgroupSockHiMaps` and `KmeshCgroupSockLoMaps`  are 2 different type, and it is not a good idea to edit codes generated by `bpf2go`. So `GetxxxMap()` is a wrapper to resolve this.